cli.js

```diff
@@ -8,6 +8,12 @@ const cli = meow(`
 	Usage
 	  $ package-json <name> [version=latest]
 
+	Options
+	  --full-metadata  Output full package metadata
+	  --all-versions   Output all versions
+	  --registry-url   Registry URL                  [Default: inferred]
+	  --no-deprecated  Omit deprecated versions      [Default: included]
```

Comment:
```suggestion
	  --no-deprecated  Omit deprecated versions
```

I feel like this is Moto as it's clear what the default is.